python/src/scipp/compat/mantid.py

```diff
@@ -110,6 +110,20 @@ def make_variables_from_run_logs(ws):
                                                       unit=sc.units.ns)
                                       })
             yield property_name, sc.Variable(data_array)
+        elif not np.isscalar(values):
+            # If property is multi-valued, create a DataArray index as coord.
+            # Note we store as DataArray in scalar Variable to prevent
+            # interference with global dimensions for for output Dataset.
```

Comment:
Why create a fake coord? What is wrong with
```python
yield property_name, sc.Variable(value=property_data)
```

Reply:
I did think about that. I think this is indeed the correct approach.